### ISSUE TITLE
Implement a Level System

### ISSUE THREAD

**Issue by torjacob** [vr4vet/Blue-Sector#27]
An increase in level means that:
- Fish get hungry more quickly
- Fish are full for less time
- Fish die more quickly
- Bigger penalty for any of the negative point givers

**Comment by fominykh** [comment]
Do you mean the game levels? If yes, we have a methodological concept for two levels:
Level 1: get introduced to the workplace, little challenge and little information given. Target: school and young job seekers.
Level 2: get more insight to the workplace, including basic training, more challenge (more realism), more information given. Target: those who are already interested in the profession and/or in the training program for it.

**Comment by torjacob** [comment]
> Do you mean the game levels? If yes, we have a methodological concept for two levels: Level 1: get introduced to the workplace, little challenge and little information given. Target: school and young job seekers. Level 2: get more insight to the workplace, including basic training, more challenge (more realism), more information given. Target: those who are already interested in the profession and/or in the training program for it.

The original intent behind this issue is to implement a system to increase difficulty when certain thresholds are met. The end goal would be unlocking the choice of harder difficulty/level when the requirements for the previous one are fulfilled.

This would be agnostic of target group, i.e. both **_Group 1_** and **_Group 2_** could experience higher difficulty over increasing levels.

Is a solution where the user states their use case / to which group they belong before starting the scenario a viable option?

If so the two groups could have a different starting difficulty.
In addition we could look into making the scenario more realistic for **_Group 2_** with more in-depth tutorial and controls, and/or more gamified / arcade based for **_Group 1_**.

In essence it would be a _"realism mode"_ and an _"arcade mode"_

As it stands this isn't something we could implement during the current sprint, but definitely something we eventually could look into if it's desirable and of a high enough priority.

**Comment by fominykh** [comment]
Both our target user groups should have the access to both levels.
We are considering locking level 2 until level 1 is successfully completed.
We think that we can use the mechanic of spending the Score/points (or any other resource) to unlock level 2. Or to create "Skill-level" requirement. And to prove that the user has a certain skill, some tasks need to be completed.

**Comment by torjacob** [comment]
closed by PR #49
with future improvements to be moved to new issue
